pages/Tin-Ingot.md

```diff
@@ -0,0 +1,10 @@
+
+## Obtaining
+Smelting one [Tin Dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Tin-Dust) in a [Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Smeltery) or an [Electric Ingot Factory](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ingot-Factory) yields one Tin Ingot.<br>
+
+Tin Ingots can be turned back to [Tin Dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Tin-Dust) by an [Electric Ingot Pulverizer](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ingot-Pulverizer)
+
+## Usage
+Tin Ingots are used in the following recipe:
+* [Tin Can](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Tin-Can)
+
```

Comment:
Should probably also mention that it is very important for alloys such as Bronze.

Reply:
For Bronze, Tin Dust is used, not the Ingot version